packages/design-system/src/components/LinkAsButton/LinkAsButton.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { cloneElement, forwardRef, Ref, useCallback } from 'react';
+import classnames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import Clickable, { ClickableProps } from '../Clickable';
+import { Icon } from '../Icon/Icon';
+import { LinkComponentProps } from '../Link';
+
+import sharedLinkableStyles from '../Linkable/LinkableStyles.module.scss';
+import linkStyles from '../Link/Link.module.scss';
+
+type LinkAsButtonProps = Omit<ClickableProps, 'className'> &
+	Omit<LinkComponentProps, 'hideExternalIcon'>;
+
+const LinkAsButton = forwardRef(
+	(
+		{ disabled, title, icon, children, ...rest }: LinkAsButtonProps,
+		ref: Ref<HTMLButtonElement>,
+	) => {
+		const { t } = useTranslation();
+
+		const getTitle = useCallback(() => {
+			if (disabled && title) {
+				return t('LINK_DISABLED_TITLE', {
+					title,
+					defaultValue: '{{title}} (this link is disabled)',
+				});
+			}
+			if (disabled) {
+				return t('LINK_DISABLED', {
+					defaultValue: 'This link is disabled',
+				});
```

Comment:
You can pass the default value directly as a string in the 2nd argument

```suggestion
				return t('LINK_DISABLED', 'This link is disabled');
```